integreat open-source-sabbatical (#270)
it-at-m/opensource.muenchen.de: software/integreat.md

LHM has been using Integreat as a software-as-a-service since February 2020.
The Integreat Munich platform provides refugees and new immigrants with multilingual initial information to help them arrive in Munich and welcome them - both via the free and offline Integreat app (download via Google Play Store or App Store) and at [integreat.app/muenchen/](https://integreat.app/muenchen/).
The overview of Munich's offerings is also a great help for full-time staff and volunteers.

## Open Source Sabbatical

We awarded the first [Open Source Sabbatical](../sabbatical) in the first half of 2025 to the [integreat-chat](https://github.com/digitalfabrik/integreat-chat) project to integrate a chat service into the Integreat app.  

> The integreat-chat uses a large language model that is trained with the existing content of the Integreat app.

Reviewer: ```suggestion
The integreat-chat uses a large language model to answer questions based on the content of the Integreat app.
```
